flink-doris-connector/src/main/java/org/apache/doris/flink/deserialization/DorisJsonDebeziumDeserializationSchema.java

```diff
@@ -63,7 +64,9 @@ public void deserialize(SourceRecord sourceRecord, Collector<String> collector)
 
     private JsonNode convertToJson(Schema schema, Object value) throws DorisException {
         if (value == null) {
-            if (schema == null) // Any schema is valid and we don't have a default, so treat this as an optional schema
+            if (schema
+                    == null) // Any schema is valid and we don't have a default, so treat this as an
+            // optional schema
```

Comment:
It feels a bit strange, or put the comment above the if?

Reply:
sure

Reply:
sure